api/internal/utils/utils_test.go

```diff
@@ -42,3 +42,33 @@ func TestSumIPs_with_nil(t *testing.T) {
 	total := sumIPs(nil)
 	assert.Equal(t, uint16(0), total)
 }
+
+func TestGenLabelMap(t *testing.T) {
+	mp := GenLabelMap("l1")
```

Comment:
need test cases for illegal parameters, such as `,` ,  `,l2:,`

Reply:
Fixed.